cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,21 @@
+import os
+import socket
+import time
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Stop app before db isn`t done"""
+
+    def handle(self, *args, **kwargs):
+        port = int(os.environ["POSTGRES_PORT"])
+        db_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+        while True:
+            try:
+                db_socket.connect(("db", port))

```

Comment:
"db" is the hardcode value for the database host. It's not a good idea because if user has another dp port this command will nor work. You must get this value from environment variable or SETTINGS.